recipes/fltk/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "fltk" %}
+{% set version = "1.3.4" %}
+{% set longversion = "1.3.4-2" %}
+{% set md5 = "69128be72365cbedd1d380739f0164e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ longversion }}-source.tar.gz
+  url: http://fltk.org/pub/{{ name }}/{{ version }}/{{ name }}-{{ longversion }}-source.tar.bz2
+  md5: {{ md5 }}
+  patches:
+    - macros.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - zlib
+    - jpeg
+    - libpng
+  run:
+    - zlib
+    - jpeg
+    - libpng
+
+test:
+  commands:
+    - echo "Tests have already been executed after compilation."
```

Comment:
Can you add tests like, https://github.com/conda-forge/bzip2-feedstock/blob/master/recipe/meta.yaml#L47-L55 ?

Reply:
Yes sure next commit :)